components/textfield/index.css

```diff
@@ -345,11 +345,15 @@
 	grid-row: 2;
 	grid-column: 1 / span 2;
 
+	/* prevents unexpected color changes for border in high contrast mode */
+	forced-color-adjust: none;
```

Comment:
Forces the input to use the styles from CSS and not the default, so that we won't get that border flash. But since WHCM is now relying on the styles set for the input, we have to set the highcontrast background color or it turns white in WHCM.